### PR TITLE
Aj/bit bucket highlight file in pr

### PR DESCRIPTION
**Pull request by ajayVibinex**

### CODE REVIEW

File: scripts/contentScript.js [hunk 1]
```diff
@@ -245,106 +245,165 @@ function addCssElementToBitbucket(highlightedPRIds, userId) {
...
+			if (elements.length == 1) {
+				let filename = elements[0].innerHTML.trim();
+				const hashedFilename = await sha256(filename);
+				if (encryptedFileNames.has(hashedFilename)) {
+					item.style.backgroundColor = '#7a7e00';
+				}
+			}
+		})
+	}
+}
```
Comment (MuskanPaliwal) [inline scripts/contentScript.js:284]: Does this change fix `highlighting files in a PR tree` problem?

File: scripts/contentScript.js [hunk 2]
```diff
@@ -245,106 +245,165 @@ function addCssElementToBitbucket(highlightedPRIds, userId) {
...
+		const fileNav = document.querySelector('[aria-label="File Tree Navigation"]');
+		if (!fileNav) return;
+		const fileList = Array.from(fileNav.getElementsByTagName('li'));
+		fileList.forEach(async (item) => {
+			let elements = item.getElementsByClassName('ActionList-item-label');
+			if (elements.length == 1) {
+				let filename = elements[0].innerHTML.trim();
+				const hashedFilename = await sha256(filename);
+				if (encryptedFileNames.has(hashedFilename)) {
+					item.style.backgroundColor = '#7a7e00';
```
Comment (MuskanPaliwal) [inline scripts/contentScript.js:279]: Why are we using hardcoded color codes?

File: scripts/contentScript.js (hunk 1 above)
Comment (ajayVibinex) [reply scripts/contentScript.js:284]: Yes

File: scripts/contentScript.js (hunk 2 above)
Comment (ajayVibinex) [reply scripts/contentScript.js:279]: has only one file, and all colors are different in each function. But still we can define all variables on top.

Comment (avikalpg) [reply scripts/contentScript.js]: Good note, @MuskanPaliwal. Creating a `tech debt` ticket to fix this in the future.

**Review by avikalpg — APPROVED** [review]

**Pull request merged**